.gitignore

```diff
@@ -20,3 +20,4 @@ tmp
 
 # vscode
 .vscode
+Gemfile.lock
```

Comment:
Let's keep `Gemfile.lock` as the version in GitHub Actions changes, and we may fail to compile the website.